cyvlfeat/fisher/cyfisher.pyx

```diff
@@ -20,34 +20,57 @@ from cyvlfeat._vl.fisher cimport VL_FISHER_FLAG_FAST
 
 
 @cython.boundscheck(False)
-cpdef cy_fisher(float[:, :] X,
-                float[:, :] MEANS,
-                float[:, :] COVARIANCES,
-                float[:] PRIORS,
+cpdef cy_fisher(np.ndarray X,
```

Comment:
I would prefer that these were left as memory views as it allows the code to be used with systems other than Numpy. It's particularly useful if someone wants to utilise the `cpdef` and thus call this method from other Cython code without resorting to numpy.